components/page-component/PageComponent.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <article>
     <header>
+      <div v-bind:class="page.theme+' colorband'"></div>
```

Comment:
i'd do this with a `header::before`, and with a bem selector ;-) `colorband--${page.theme.slug}`